lib/interpreters/multi_trips.rb

```diff
@@ -22,23 +22,31 @@ module Interpreters
   class MultiTrips
     def expand(vrp)
       vrp.vehicles = vrp.vehicles.collect{ |vehicle|
-        if vehicle.trips && vehicle.trips >= 1
-          new_ids = (0..vehicle.trips - 1).collect{ |index| vehicle.id + '_trip_' + index.to_s }
-          containing_relations = vrp.relations.select{ |relation| relation.linked_vehicle_ids.include?(vehicle.id) }.each{ |relation|
-            relation.linked_vehicle_ids -= vehicle.id
-            relation.linked_vehicle_ids += new_ids
-          }
-          vrp.relations += [Models::Relation.new(type: 'vehicle_trips', linked_vehicle_ids: new_ids)]
+        return vehicle unless vehicle.trips && vehicle.trips > 1
```

Comment:
This looks like a bug :thinking: `return` returns from the function `expand` but if I understand correctly your intention was to collect the vehicles in an array without duplicating them if they don't satisfy the condition. I think the if condition was the right way.

If this didn't break any tests, this means we don't have unit test for `MultiTrips.expand()`. Maybe we should add a test (which fails in this state of the code) with "mixed" vehicles (i.e., with vehicle.trips = {nil, 1 and 2}) that checks that we return 4 vehicles and relations are there, etc..